GameOfLife/Displaying/GameIterator.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    public class GameIterator
```

Comment:
So, why is GameIterator printing things? Based on naming, i would have guessed it was the one updating cells to next generation. If its job is to print things, then the naming should be clear about it

Reply:
Changed name to DisplayGameField